packages/common/src/components/NbCondaStore.tsx

```diff
@@ -0,0 +1,44 @@
+import { INotification } from 'jupyterlab_toastify';
+import { CondaStore } from '..';
+import {NbConda} from './NbConda'
+
+
+export class NbCondaStore extends NbConda {
+    async loadEnvironments(): Promise<void> {
+        if (!this.state.isLoading) {
+            this.setState({isLoading: true})
+            try {
+                const environments = await CondaStore.fetchEnvironments()
+                this.setState({...this.state, isLoading: false, environments})
+            } catch (error) {
+                if (error !== 'cancelled') {
+                    console.error(error)
+                    INotification.error(error.message)
+                    this.setState({isLoading: false})
```

Comment:
Question: Is there a way to check the availability of conda-store environments before this loads. i.e maybe catch this beforehand and grey out `Conda Store Packages Manager` in Settings?

Reply:
I know there's a way of greying out options (because I can see greyed out options in a JupyterLab menu right now...) but I'm not sure how to do it. I agree, it might be cool to fire off a request to `localhost:5000` just to see if conda-store is running, and then set the state of the dropdown menu based on that.

Reply:
Conda Store may be deployed on another server, and another port than 5000. 

As we can't ship with an hardcoded URL, I see two options : 
- For the JupyterLab extension itself, we'll need a place in the UI to setup the API endpoint of Conda Store.
- For the standalone version (which actually spins a JupyterLab with the extension), we may pass the URL as an option in the command line.

This change will be necessary for the first phase of the project, but keep using `localhost:5000` as a constant for the moment, the time we figure out other options and their impacts.

I'm adding an issue to address this specific point and we'll discuss all this during our next meeting :)